forgerock-authenticator/build.gradle

```diff
@@ -96,15 +96,19 @@ dependencies {
     implementation 'com.nimbusds:nimbus-jose-jwt:9.0.1'
 
     // Common
-    implementation 'androidx.annotation:annotation:1.1.0'
+    implementation 'androidx.annotation:annotation:1.3.0'
```

Comment:
Not sure if we ever update the copyright notes with current year for build.gradle...?